repo/repoWin.ps1

```diff
@@ -1,25 +1,33 @@
-﻿
-# Script for running 'repo' tool in either Windows Subsystem for Linux's or Cygwin's bash
-
-Param($command, $drive, $path)
-
-if (Test-Path $path) {
-    $path = ($path -replace '\\','/')
-    $path = ($path -replace '.:',"/$drive/c")
-    $cmdPath = (Split-Path -parent $PSCommandPath)
-
-    if ($drive -ne "cygdrive") {
-        $cmdPath = $($cmdPath -replace '\\','/')
-        $cmdPath = ($cmdPath -replace '.:',"/$drive/c")
-
-        bash -l "$cmdPath/repo" $command -f $path
-    } 
-    else {
-        C:\cygwin64\bin\bash.exe -l "$cmdPath\repo" $command -f $path
-    }
-}
-else {
-    Write-Error "File $path not found"
-    exit 1
-}
-Write-Output "Executed $command on $path"
\ No newline at end of file
+﻿
+# Script for running 'repo' tool in either Windows Subsystem for Linux's or Cygwin's bash
+
+Param($command, $path)
+
+if (Test-Path $path) {
+    $cmdPath = Split-Path -parent $PSCommandPath
+
+    #if Windows Subsystem for Linux (WSL)
+    if ( $(Get-Command bash).Source -eq "C:\WINDOWS\system32\bash.exe") {
+        $path = $path -replace '\\','/'
+        $path = $path -replace '.:',"/mnt/c"
+
+        $cmdPath = $cmdPath -replace '\\','/'
+        $cmdPath = $cmdPath -replace '.:','/mnt/c'

```

Comment:
Is there a reason why this logic can't be in `repo` too?

Reply:
Putting this code in `repo` results in stripping all backslashes. For `path`:
`repo: does not exist: C:DevgitDigitalPlatformuisrcmaincontentjcr_rootappschipotlecomponentsstructurepagev1pagepartialsgascript.html`
for `cmdPath`:
`/bin/bash: C:githubtoolsrepo/repo: No such file or directory`